# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/gtest/TestMP3Demuxer.cpp
# Commit: b6c7c93d0406
# Full Hash: b6c7c93d040640176d198631d41e87a056f80265
# Author: alwu <alwu@mozilla.com>
# Date: 2023-08-02 09:45:13
# Regressor Bug: 1845213
# File Overlap Count: 1
# Description:
#   Bug 1845213 - part2 : round up the inbase to make the result more precise. r=padenot
#   
#   Eg. {448/48,000} - {1/1,000,000} without rounding, the result would be
#   {447/48,000}, which loses around 20ms precision.
#   
# ==============================================================================

diff -r 5a45000b88bb -r b6c7c93d0406 dom/media/gtest/TestMP3Demuxer.cpp
--- a/dom/media/gtest/TestMP3Demuxer.cpp	Wed Aug 02 00:19:07 2023 +0000
+++ b/dom/media/gtest/TestMP3Demuxer.cpp	Wed Aug 02 00:19:07 2023 +0000
@@ -106,7 +106,7 @@
       // The tolerance comes from the fact that this file has ID3v1 information
       // at the end, this trips our CBR duration calculation. The file has
       // however the correct duration when decoded / demuxed completely.
-      res.mDuration = Some(MP3Resource::Duration{30093062, 0.00015f});
+      res.mDuration = Some(MP3Resource::Duration{30093063, 0.00015f});
       res.mSeekError = 0.02f;
       res.mSampleRate = 44100;
       res.mSamplesPerFrame = 1152;